### PR TITLE
Feature/91 additional nginx ingress annotations in dogu resource

### LINKED ISSUE #91: Include additional nginx-ingress annotations in dogu resource
It should be possible to add nginx-ingress annotations to the dogu resource which in turn get added to the dogu's nginx-ingress rule.

This is necessary, because for example Nexus needs a bigger `client_max_body_size`. To ensure further extensibility and flexibility, a new field for nginx-ingress annotations should be added to the dogu resource.

### PR DESCRIPTION
**Pull request by jelemux**
Resolves #91

### CODE REVIEW

**Comment by nhinze23** [comment]
Documentation is missing.

File: controllers/annotation/ingressAnnotator.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+// AppendIngressAnnotationsToService marshals the additional ingress annotations to json and adds them to the service as an annotation with the key of AdditionalIngressAnnotationsAnnotation.
+// These annotations are then to be read by the service discovery and appended to the ingress object for the dogu.
+func (i IngressAnnotator) AppendIngressAnnotationsToService(service *corev1.Service, additionalIngressAnnotations doguv1.IngressAnnotations) error {
+	err := appendAdditionalIngressAnnotations(service, additionalIngressAnnotations)
+	if err != nil {
+		return fmt.Errorf("failed to append annotation [%s] to service [%s]: %w", AdditionalIngressAnnotationsAnnotation, service.GetName(), err)
+	}
+
+	return nil
+
```
Comment (nhinze23) [inline controllers/annotation/ingressAnnotator.go:26]: Delete empty line

Comment (nhinze23) [inline controllers/annotation/ingressAnnotator.go:26]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,10 +5,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- [#91] Add additional ingress annotations to dogu resource.
```
Comment (jelemux) [inline CHANGELOG.md:9]:
```suggestion
- [#91] Add additional ingress annotations to dogu resource.
- [#91] Append those annotations to the dogu's service.
```

**Review by nhinze23 — APPROVED** [review]
lgtm

**Pull request merged**